src/node_api_embedding.h

```diff
@@ -0,0 +1,234 @@
+#ifndef SRC_NODE_API_EMBEDDING_H_
+#define SRC_NODE_API_EMBEDDING_H_
```

Comment:
Add a warning here that it is not promised to be ABI stable.

Reply:
The warning is added.